charts/metrics-server/README.md

```diff
@@ -22,55 +22,55 @@ helm upgrade --install metrics-server metrics-server/metrics-server
 
 The following table lists the configurable parameters of the _Metrics Server_ chart and their default values.
 
-| Parameter                            | Description                                                                                                                                                                                                                                                      | Default                                    |
-| ------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------ |
+| Parameter                            | Description                                                                                                                                                                                                                                                      | Default                                         |
```

Comment:
It's nice that GitHub UI display whitespace diff correctly now unlike in my other PR 
![image](https://user-images.githubusercontent.com/767257/197305138-6ff5791d-6ea0-486f-86c2-b98fb96154ae.png)